homeassistant/components/plaato/entity.py

```diff
@@ -0,0 +1,85 @@
+"""PlaatoEntity class."""
+from pyplaato.models.device import PlaatoDevice
+
+from homeassistant.helpers import entity
+
+from .const import DEVICE, DEVICE_ID, DEVICE_NAME, DEVICE_TYPE, DOMAIN, SENSOR_DATA
+
+
+class PlaatoEntity(entity.Entity):
```

Comment:
```suggestion
class PlaatoEntity(CoordinatorEntity):
```


from homeassistant.helpers.update_coordinator import CoordinatorEntity